src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoLogin.java

```diff
@@ -0,0 +1,120 @@
+package it.pn.frontend.e2e.api.destinatario;
+
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+import org.apache.hc.core5.http.ClassicHttpRequest;
+import org.apache.hc.core5.http.HttpEntity;
+import org.apache.hc.core5.http.HttpHeaders;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.apache.hc.core5.http.io.support.ClassicRequestBuilder;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.junit.Assert;
+
+import java.io.IOException;
+public class SpidDemoLogin {
+
+    private static final Logger logger = LoggerFactory.getLogger("SpidDemoLogin");
+    private String spidDemoLoginEndPoint;
+    private String responseBody;
+
+    /*
+    parametri di input
+     */
+
+    private final String username;
+    private final String password;
+    private final String spidLevelInput;
+    private final String organizationDisplayNameInput;
+    private final String samlRequestInput;
+    private final String relayStateInput;
+    private final String sigAlgInput;
+    private final String signatureInput;
+    private final String purposeInput;
+    private final String minAgeInput;
+    private final String maxAgeInput;
+    private final String retryInput;
+
```

Comment:
miss comment to end input as in src/test/java/it/pn/frontend/e2e/api/destinatario/SpidDemoStart.java

Reply:
done